### PR TITLE
LUN-220: Fix multiline style issue

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

File: src/routes/DisciplineSelectionScreen.tsx [hunk 1]
```diff
@@ -101,14 +101,16 @@ const DisciplineSelectionScreen = ({ route, navigation }: DisciplineSelectionScr
...
       <StatusBar backgroundColor='blue' barStyle='dark-content' />
-      <Title
-        title={discipline.title}
-        description={`${discipline.numberOfChildren} ${
-          discipline.numberOfChildren === 1 ? labels.home.unit : labels.home.units
-        }`}
-      />
       <ServerResponseHandler error={error} loading={loading} refresh={refresh}>
         <StyledList
+          ListHeaderComponent={
```
Comment (klinzo) [inline src/routes/DisciplineSelectionScreen.tsx:106]: Why this change?

**Review by klinzo — APPROVED** [review]
Tested on android

File: src/routes/DisciplineSelectionScreen.tsx (hunk 1 above)
Comment (SteffiStuffel) [reply src/routes/DisciplineSelectionScreen.tsx:106]: Otherwise the header would not scroll so you have a fixed header always visible, and the ScrollArea of the StyledList ist beneath.

**Review by Taggotty — APPROVED** [review]
Tested on Android emulator
